docs/application/web/guides/connectivity/download.md

```diff
@@ -85,25 +85,45 @@ To provide the user access to Internet resources, you must learn how to manage d
 
    The `start()` method returns a unique identifier for the download operation.
 
-5. During the download:
+   You can check the status of the download operation by calling the `getState()` method with the download ID:
+
+      ```
+      tizen.download.getState(downloadId);
+      ```
+
+5. During the download operation, you can modify the download state.
+
+    >**Note**
+    >
+    >Depending on the download size and the connection speed, the download operation completes almost instantly.
```

Comment:
...  the download operation could complete almost instantly.  - sounds better for me.

Reply:
@pkosko  we avoid using future tenses as a part of thumb rule for Tizen documentation. That's why we must not use (could, would, will) or any word that represents future tense. 

Reference: [Google Style guide](https://developers.google.com/style/tense)

Reply:
@nawab-samsung then I suggest rephrase like below:

For downloading small files and high connection speed, the download operation completes almost instantly.

Reply:
After discussion with Piotr Koska we decided to change it to:

"For downloading small file with using high speed connection, the download operation completes almost instantly."

@nawab-samsung please let us know if that's ok.

Reply:
Slight change
 For downloading small files using the high-speed connection, the download operation completes almost instantly.